src/main/java/org/jointheleague/features/Derek/Praise.java

```diff
@@ -0,0 +1,40 @@
+package org.jointheleague.features.Derek;
+
+import org.javacord.api.event.message.MessageCreateEvent;
+import org.jointheleague.features.examples.first_features.abstract_classes.Feature;
+import org.jointheleague.features.help_embed.plain_old_java_objects.help_embed.HelpEmbed;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class Praise extends Feature {
+
+    public final String COMMAND = "!praise";
+
+    public Praise(String channelName) {
+        super(channelName);
+
+        //Create a help embed to describe feature when !help command is sent
+        helpEmbed = new HelpEmbed(
+                COMMAND,
+                "I thought the name says it all. It gives praise. That's it. Stop asking"
+        );
+    }
+
+    @Override
+    public void handle(MessageCreateEvent event) {
+        String messageContent = event.getMessageContent();
+        if (messageContent.startsWith(COMMAND)) {
+            ArrayList<String> list = new ArrayList<String>();
```

Comment:
Inefficient, the declaration should probably be moved out of the function and the .add calls should be moved to the constructor